Dairy/Dairy/IPriorityTask.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Dairy
+{
+    internal interface IPriorityTask:IRegularTask
+    {
+        public Priority TaskPriority { get; }
```

Comment:
drop access modifier
```suggestion
        Priority TaskPriority { get; }
```